src/lib.rs

```diff
@@ -51,25 +51,22 @@
 // FIXME(waffle): use `docsrs` here when issue with combine is resolved <https://github.com/teloxide/teloxide/pull/305#issuecomment-716172103>
 #![cfg_attr(all(teloxide_docsrs, feature = "nightly"), feature(doc_cfg))]
 
-pub use bot::{Bot, BotBuilder};
 pub use dispatching::repls::{
     commands_repl, commands_repl_with_listener, dialogues_repl, dialogues_repl_with_listener, repl,
     repl_with_listener,
 };
-pub use errors::{ApiErrorKind, DownloadError, KnownApiErrorKind, RequestError};
+pub use teloxide_core::{ApiError, DownloadError, RequestError};
 
-mod errors;
-mod net;
+mod logging;
 
-mod bot;
 pub mod dispatching;
 pub mod error_handlers;
-mod logging;
 pub mod prelude;
-pub mod requests;
-pub mod types;
 pub mod utils;
 
+pub use teloxide_core as core;
```

Comment:
I think it would be better to just reexport everything, so users wouldn't even need to worry about `core`.

```rust
#[doc(inline)]
pub use teloxide_core::*;
```
Maybe?